userspace/falco/app_cmdline_options.cpp

```diff
@@ -181,6 +181,7 @@ void cmdline_options::define()
 #endif
 		("M",                             "Stop collecting after <num_seconds> reached.", cxxopts::value(duration_to_tot)->default_value("0"), "<num_seconds>")
 		("markdown",                      "When used with --list/--list-syscall-events, print the content in Markdown format", cxxopts::value<bool>(markdown))
+		("monitor_files",                 "Monitor rules and config files to reload Falco on change.", cxxopts::value<bool>(monitor_files))
```

Comment:
It would be preferable to have a config value and not a command-line flag option. Wdyt?

Reply:
I agree!